kuma/static/styles/components/content.styl

```diff
@@ -7,6 +7,13 @@
 
 $table-blue = #d4dde4;
 
+/*
+    Used to draw borders around objects which should only be bordered in
+    the editor.
+*/
+
+$editor-box-border-style = solid 1px #6699FF40;
```

Comment:
Hm. .hidden is very large to put into a variable like that. Not sure how well it would work. Do you really think it's worth doing? I kinda don't.